source/standards/git.html.md.erb

```diff
@@ -18,6 +18,12 @@ developers on your project. This includes:
 * any future developers (including yourself) who want to see why a change was
   made
 
+Taking the time to write a good commit message is important for a few reasons.
```

Comment:
This still kinda duplicates the first sentence of the section, and the following para could arguably read better as an amendment to the last bullet point in the list of reasons?

Reply:
Agreed!